### ISSUE TITLE
Support Gitea repositories

### ISSUE THREAD

**Issue by agjohnson** [readthedocs/readthedocs.org#11103]
There might be some integration work, but for now I think support could also just be adding Gitea setup to our set up instructions, similar to Azure/etc.

Gitea is a fairly popular host to replace GitHub, and I believe the project aims for some amount of parity with GitHub implements (ie Actions), so it might be easier to support than GitLab. Just a guess though.

**Comment by steko** [comment]
See also https://github.com/readthedocs/readthedocs.org/issues/10888 (particularly re: Forgejo) and https://github.com/readthedocs/readthedocs.org/issues/8364

**Comment by humitos** [comment]
Closing. Read https://github.com/readthedocs/readthedocs.org/issues/8364#issuecomment-3943666543 for more.

**Comment by agjohnson** [comment]
Noted above, but this issues is more about documenting support for now, similar to Azure

>  I think support could also just be adding Gitea setup to our set up instructions, similar to Azure/etc.

I agree there isn't yet a strong case otherwise for native support.

**Comment by humitos** [comment]
We already have documentation for Gitea at https://docs.readthedocs.com/platform/stable/guides/setup/git-repo-manual.html#manual-integration-setup

**Comment by agjohnson** [comment]
Yup, but not here https://docs.readthedocs.com/platform/stable/guides/creating-project-private-repository.html

This was work I was focusing on around overhauling the integration docs.
